converter/fluka/helper_parsers/beam_parser.py

```diff
@@ -123,21 +123,17 @@ class FlukaBeam:
 def convert_energy(beam_json: dict) -> float:
     """
     Extract energy from beam JSON and provide it in Fluka convention.
-
-    For particles other than HEAVYIONS (i.e. protons, alpha particles, neutrons, electrons) MeV unit is used.
-    For HEAVYIONS (i.e. nuclei heavier than helium, defined by A and Z numbers) MeV/u is used.
-    Note than MeV/u (atomic mass unit) is not the same as MeV/nucl (number of nucleons).
-    For more details see:
-    https://flukafiles.web.cern.ch/manual/chapters/description_input/description_options/beam.html#beam.
```

Comment:
I think its essantial to keep that link to the FLUKA manual.
Its important to stress that this a different convention that used in SHIELD-HIT12A.

If I read it correctly SHIELD-HIT12A uses MeV/nucl for helium-4, while FLUKA uses MeV for helium-4.

Its also worth noting here that FLUKA expects in fact GeV, not MeV and we will use factor 1000 when converting from user inputted.